src/dotnet/commands/dotnet-list/dotnet-list-package/LocalizableStrings.resx

```diff
@@ -0,0 +1,171 @@
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="AppFullName" xml:space="preserve">
+    <value>List all package references of the project.</value>
+  </data>
+  <data name="CmdFramework" xml:space="preserve">
+    <value>FRAMEWORK | FRAMEWORK\RID</value>
+  </data>
+  <data name="CmdFrameworkDescription" xml:space="preserve">
+    <value>Chooses a framework to show its packages. Use the option multiple times for multiple frameworks.</value>
+  </data>
+  <data name="CmdOutdatedDescription" xml:space="preserve">
+    <value>Lists packages that have newer versions.</value>
+  </data>
+  <data name="CmdTransitiveDescription" xml:space="preserve">
+    <value>Lists transitive and top-level packages.</value>
+  </data>
+  <data name="MultipleProjectsFound" xml:space="preserve">
+    <value>The directory given includes multiple projects. Please specify one of the following projects:</value>
+  </data>
+  <data name="MultipleSolutionsFound" xml:space="preserve">
+    <value>The directory given includes multiple solutions. Please specify one of the following solutions:</value>
+  </data>
+  <data name="NoProjectsOrSolutions" xml:space="preserve">
+    <value>The given directory does not include any solution or project.</value>
+  </data>
+  <data name="CmdConfig" xml:space="preserve">
+    <value>CONFIG FILE</value>
+  </data>
+  <data name="CmdConfigDescription" xml:space="preserve">
+    <value>A path to a config file to specify sources. Works only with `--outdated`.</value>
+  </data>
+  <data name="CmdHighestMinorDescription" xml:space="preserve">
+    <value>Considers only the versions that have matching major when looking for latest version. Works only with `--outdated`.</value>
+  </data>
+  <data name="CmdHighestPatchDescription" xml:space="preserve">
+    <value>Considers only the versions that have matching minor and major when looking for latest version. Works only with `--outdated`.</value>
+  </data>
+  <data name="CmdPrereleaseDescription" xml:space="preserve">
+    <value>Considers prerelease versions when looking for latest version. Works only with `--outdated`.</value>
+  </data>
+  <data name="CmdSource" xml:space="preserve">
+    <value>SOURCE</value>
+  </data>
+  <data name="CmdSourceDescription" xml:space="preserve">
+    <value>Sources to lookup for latest versions. Works only with `--outdated`.</value>
+  </data>
+  <data name="OutdatedOptionOnly" xml:space="preserve">
+    <value>This option `{0}` can only be used with `--outdated` present.</value>
+  </data>
+  <data name="FileNotFound" xml:space="preserve">
+    <value>The given file was not found. Please make sure that you are passing the correct path.</value>
```

Comment:
We have a similar error message to this one when no project files could be found.  Perhaps `Could not find file or directory '{0}'.`?